submissions/career/career.md

```diff
@@ -0,0 +1,58 @@
+
+### Correlation vs Causation
+
+In writing this piece,  I made a lovely little [post](https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/) on the r/Factorio subreddit, asking people about their relationship with Factorio and STEM, and how it has inspired/changed their education and careers. As always with these kinds of things, I got some of what I wanted, and some interesting new perspectives I hadn't really considered.
+
+> At 32 years old I started my STEM career before Factorio was created, and way before I started playing the game. There is definitely a correlation between the kind of mind that goes into STEM and the kind of mind required to play and enjoy this game.
+*u/Bigtallanddopey - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxiwjwy?utm_source=share&utm_medium=web2x&context=3*
+
+> I think it's a correlation. People with STEM aspirations tend to enjoy/be attracted more to games like Factorio. The mindset is typically already there, or else they don't like it and stop playing it.
+*u/Vexinimous - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxif1dj?utm_source=share&utm_medium=web2x&context=3*
+
+These two comments raise very eloquently the idea that Factorio is an *engineer/scientist/mathematician*'s dream come true. An open playground with clearly defined rules to explore, design and create interesting stuff with no upfront material cost, all whilst sat in our comfy chairs. It's an opportunity for people to explore their creativity and stretch and expand that *oh so special* muscle in the brain responsible for critical thinking and design.
+
+### Nerd sniping for jobs...?
+
+Around these parts, we love [nerd sniping](https://xkcd.com/356/) more than most (see our [fully interactive spidertron](https://alt-f4.blog/ALTF4-12/#building-spidertron-for-the-web-xthexder)). Delving into the pointless questions, not because the answers are important, but because the problem is interesting and satisfying to work on. *TheXtrafresh* talks about being hired in IT, using ~~pointless~~ interesting Factorio shenanigans to help get them in. And of course, we can't forget to mention the actual [scientific papers](https://arxiv.org/abs/2102.04871) written around Factorio.
+
+> I applied for a job in IT integration at 36 years old, without training, certificates, experience or anything else relevant. I showed them this topic of a nicely pointless overcomplication of smelting: https://forums.factorio.com/viewtopic.php?p=319620#p319620 I still work there :)
+*u/TheXtrafresh - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxj9g1x?utm_source=share&utm_medium=web2x&context=3*
+
+## Conclusion
+
+The key parts of Factorio and other [logistics, base building, sandbox, puzzle, problem-solving, real-time strategy](https://www.reddit.com/r/factorio/comments/9z8x5m/does_factorio_merit_the_creation_of_a_new/) games are:
```

Comment:
This is still a little long-winded. I don't think a conclusion is the right place for a bulleted list of info :P You can maybe rework it to be in a single sentence, and then you reference only a few of those points.